File: algorithms/Codility-master/Codility-master/6_Sorting/MaxProductOfThree.py
#  100%
def solution(A):
    # write your code in Python 3.6
    if len(A) >=3:
        A.sort()
        return max(A[-3] * A[-2] * A[-1] , A[0] * A[1] * A[-1])
    else:
        return 0

File: algorithms/Codility-master/Codility-master/6_Sorting/MaxProductOfThree.js
// 100%
function solution(A) {
    // write your code in JavaScript (Node.js 8.9.4)
    if( A.length == 0){
        return 0;
    }else{
        let B = A.sort((a, b) => b - a);
        let x = Math.max((B[0] * B[B.length-2] * B[B.length-1]) ,( B[0] * B[1] * B[2]));
        return x;
    }
}
